src/bucket.js

```diff
@@ -71,6 +81,7 @@ export default class Bucket {
    * @param  {Object} options.headers The headers object option.
    * @return {Promise<Object, Error>}
    */
+  @nobatch("Batch doesn't support read operations.")
```

Comment:
> you won't be able to consume these results before having actually sent the requests asking for them

Yes I am aware of that.

If you just want to retrieve data and you do not need to process this data to add request to the batch then you can just do what you said:

```
api.bucket("blog").batch(batch => {
    batch.collections('articles').get_records(id="e9fe125c-e865-11e5-bbab-04dad2bf2cac");
    batch.collections('comments-e9fe125c-e865-11e5-bbab-04dad2bf2cac').get_records();
})
.then((batchResults) => {
    // oh my, it's here I'm getting the read operation results, not above where I really needed them!
    const articleResponse = batchResults[0];
    const commentsResponse = batchResults[1];
  });
```